gmso/core/angle_type.py

```diff
@@ -69,6 +69,15 @@ def __init__(
     def member_types(self):
         return self.__dict__.get("member_types_")
 
+    def __repr__(self):
+        """Return a formatted representation of the bond type."""
+        desc = (
+            f"<{self.__class__.__name__} {self.name},\n "
+            f"expression: {self.expression},\n "
+            f"id: {id(self)}>,\n"
+            f"member types: {self.member_types}"
+        )
```

Comment:
Piggybacking off of my comment
```suggestion
    def __repr__(self):
        """Return a formatted representation of the bond type."""
        desc = super().__repr__()
        desc = desc.replace(">", f", \n member types: {self.member_types}>")
        )
```

Reply:
Good point